tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -14,6 +14,94 @@ namespace Microsoft.Data.Analysis.Tests
 {
     public partial class DataFrameTests
     {
+        internal static void VerifyColumnTypes(DataFrame df, bool testArrowStringColumn = false)
```

Comment:
This is duplicated both in M.D.A and M.D.A.IO, since both unit test projects use this function. I found duplicating the function was the easiest work around for now.